migrations/versions/7f041b65b4ce_.py

```diff
@@ -0,0 +1,86 @@
+"""empty message
+
+Revision ID: 7f041b65b4ce
+Revises: 
+Create Date: 2017-06-28 15:55:41.212533
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+from sqlalchemy.sql import table, column, select
+import uuid
+import base64
+from progressbar import ProgressBar
+import progressbar.widgets
+
+
+revision = '7f041b65b4ce'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+user_table = table('user', column('uid', sa.String(22)), column('uuid', sqlalchemy_utils.types.uuid.UUIDType(binary=False)))
```

Comment:
Add an `id` column as well so that the updates are faster: they'll go against the primary key instead of a string index.